Sources/Optimizely/OptimizelyClient+ObjC.swift

```diff
@@ -304,13 +322,84 @@ extension OptimizelyClient {
         try self.track(eventKey: eventKey, userId: userId, attributes: attributes, eventTags: eventTags)
     }
     
+    @available(swift, obsoleted: 1.0)
```

Comment:
We need to talk about this one.  I don't feel comfortable making something like this public.  Too much chance for error.